File: dataset/human/c/file_2616.c
#include<stdio.h>
 int main(void)
 {
     char s[3];
     int i,j=0,k=0,l=0;

     scanf("%s",s);

     for(i=0;i<3;i++){
       if(s[i]=='a') j++;
       if(s[i]=='b') k++;
       if(s[i]=='c') l++;
     }

     if((j==1)&&(k==1)&&(l==1)) printf("Yes\n");
     else printf("No\n");

     return 0;
 }

File: dataset/human/java/file_2616.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {

	public static void main(String... nui) {
		
		Scanner sc = new Scanner(System.in);

		char[] s = sc.next().toCharArray();
		char[] t = sc.next().toCharArray();
		
		Arrays.sort(s);
		Arrays.sort(t);
		
		String l = new String(s);
		String r = new StringBuilder(new String(t)).reverse().toString();
		
		System.out.println((l.compareTo(r) < 0) ? "Yes" : "No");
		
	}

}
